rocAL/rocAL_pybind/amd/rocAL/fn.py

```diff
@@ -0,0 +1,534 @@
+
+from amd.rocAL import readers
+from amd.rocAL import decoders
+import inspect
+import amd.rocAL.types as types
+import rali_pybind as b
+from amd.rocAL.pipeline import Pipeline
+
+def blend(*inputs,**kwargs):
```

Comment:
Can you check for the parameter - blend?

Reply:
Added necessary params for blend